kernel/patches/4.19.x/0002-Enable-memory-hotplug-using-probe-for-arm64.patch

```diff
@@ -1,7 +1,7 @@
-From 074a6a9d83a4e790f892ef0fc91cdabbfbf26202 Mon Sep 17 00:00:00 2001
```

Comment:
with this patch, now we have 3 patches that start with `0001`, would you mind changing their name?

Reply:
I believe this change actually renames all the patches to have a proper order. Do you think i have missed it?

Reply:
oops!, yes, you're right :smile: